STABLE_RELEASES.md

```diff
@@ -0,0 +1,147 @@
+  * *Severe regressions* from a Cosmos-SDK's previous release. This includes all sort of issues
+    that may cause the core packages or the `x/` modules unusable.
+  * Bugs that may cause **loss of user's data**.
+* Other safe cases:
+  * Bugs which don't fit in the aforementioned categories for which an obvious safe patch is known.
+  * Relatively small yet strictly non-breaking changes that introduce forward-compatible client
+    features to smoothen the migration to successive releases.
+
+### What does not qualify as SRU
+
+* State machine changes.
+* New features that introduces API breakages (e.g. public functions removal/renaming).
+* Cosmetic fixes, such as formatting or linter warning fixes.
+
+## Code Owner Membership
+
+In the ethos of open source projects, and out of necessity to keep the code
+alive, the core contributor team will strive to permit special repo privileges
+to developers who show an aptitude towards developing with this code base.
+
+Several different kinds of privileges may be granted however most common
+privileges to be granted are merge rights to either part of, or the entirety of the
+code base (through the github `CODEOWNERS` file). The on-boarding process for
+new code owners is as follows: On a bi-monthly basis (or more frequently if
+agreeable) all the existing code owners will privately convene to discuss
+potential new candidates as well as the potential for existing code-owners to
+exit or "pass on the torch". This private meeting is to be a held as a
+phone/video meeting. Subsequently at the end of the meeting, one of the existing
+code owners should open a PR modifying the `CODEOWNERS` file. The other code
+owners should then all approve this PR to publicly display their support.
+
+Only if unanimous consensus is reached among all the existing code-owners will
+an invitation be extended to a new potential-member. Likewise, when an existing
+member is suggested to be removed/or have their privileges reduced, the member
+in question must agree on the decision for their removal or else no action
+should be taken. If however, a code-owner is verifiably shown to intentionally
+have had acted maliciously or grossly negligent, code-owner privileges may be
+stripped with no prior warning or consent from the member in question.
+
+Earning this privilege should be considered to be no small feat and is by no
+means guaranteed by any quantifiable metric. It is a symbol of great trust of
+the community of this project.
+
+# 0.37
+
+The `0.37.x` release series will continue receiving bugfixes until the Cosmos Hub
+migrates to a newer release of the Cosmos-SDK.
+
+# 0.38
+
+Discontinued since **0.39.0** is published.
+
+# 0.39 - The Launchpad Release Series
+
+The **0.39 Launchpad** release series will be supported until 6 months after **0.40.0** is published. A fairly strict **bugfix-only** rule applies to pull requests that are requested to be included into a stable point-release.
```

Comment:
should not launchpad be maintained longer? 0.39 is the last release with amino and it may take longer than 6 months for people to migrate.

Reply:
In line of principle I agree, we should support it at least for 1 year. The `6 months` was agreed as a tentative time limit, and that's why I don't feel comfortable in using the label `LTS` anymore. `LTS` usually indicates few years-long support timetable and this is not the case. Maybe once `0.40` is published we could reassess and extend support (again, another good reason for not labelling Launchpad **LTS**).